cmds/id/id_test.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+)
+
+var (
+	testpath = "."
+	logPrefixLength = 20
```

Comment:
one option for something like this is to say
logPrefixLength = len("2009/11/10 23:00:00 ")

https://play.golang.org/p/_4a1beGP65